package.json

```diff
@@ -1,17 +1,22 @@
 {
   "version": "1.0.0",
   "scripts": {
-    "test": "node_modules/.bin/lerna run test",
-    "lint": "node_modules/.bin/lerna run lint && node_modules/.bin/eslint './gulp/**/*.{js,jsx}' --fix",
-    "doc": "node_modules/.bin/lerna run doc",
-    "audit": "node_modules/.bin/lerna run audit",
-    "build": "node_modules/.bin/lerna run build",
-    "postinstall": "node_modules/.bin/lerna bootstrap --no-ci && node_modules/.bin/lerna run build"
+    "test": "lerna run test",
+    "lint": "lerna run lint && node_modules/.bin/eslint './gulp/**/*.{js,jsx}' --fix",
+    "doc": "lerna run doc",
+    "audit": "lerna run audit",
+    "build": "lerna run build",
+    "postinstall": "node_modules/.bin/lerna bootstrap --no-ci && node_modules/.bin/lerna run build",
+    "release": "lerna version --force-publish",
```

Comment:
Remove the --force-publish flag, it would cause all packages to be released every time you run release command.

Reply:
The flag removed.